.github/workflows/ci.yml

```diff
@@ -68,3 +68,5 @@ jobs:
         run: sudo apt install -y dotnet6
       - name: Run script to make sure all files end with EOL 
         run: dotnet fsi eofConvention.fsx
+      - name: prettier
```

Comment:
@realmarv please use a better name, see other name of steps

Reply:
`Adopt "prettier" for our TypeScript code` is a good name for a commit or PR title, but for a CI step? Once this PR gets merged, then we have adopted prettier, but it only happens once. The step "runs prettier", it doesn't adopt it.